src/components/show-products/styles.css

```diff
@@ -70,3 +70,47 @@
 .checkbox-timeFrame-30 {
   background-color: #e5f5e2;
 }
+
+.filter-form {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  margin: 0 auto 30px;
+  position: relative;
+}
+
+.search-item {
+  border-radius: 4px;
+  line-height: 2;
+  margin-top: 8px;
+  margin-bottom: 20px;
+  padding: 0px 10px;
+  width: 100%;
+}
+
+/* clears the ‘X’ from Internet Explorer */
+.search-item::-ms-clear {
+  display: none;
+  width: 0;
+  height: 0;
+}
+.search-item::-ms-reveal {
+  display: none;
+  width: 0;
+  height: 0;
+}
+/* clears the ‘X’ from Chrome */
+.search-item::-webkit-search-decoration,
+.search-item::-webkit-search-cancel-button,
+.search-item::-webkit-search-results-button,
+.search-item::-webkit-search-results-decoration {
+  display: none;
+}
```

Comment:
You can avoid overriding these styles by changing the `input[type=search]` to `type="text"`.